test/unit/command/http-command.test.ts

```diff
@@ -282,7 +282,7 @@ describe('http command', () => {
 					},
 					rawHeaders: 'test: abc',
 					rawBody: '200 Ok',
-					rawOutput: '200 Ok',
+					rawOutput: 'HTTP/null 200\ntest: abc\n\n200 Ok',
```

Comment:
The test should be improved to be more realistic here. The field should never be `null`